app/javascript/styles/components.scss

```diff
@@ -2176,7 +2176,7 @@ button.icon-button.active i.fa-retweet {
   color: $ui-primary-color;
   cursor: pointer;
   font-size: 16px;
-  padding: 0 15px;
+  padding-right: 15px;
```

Comment:
- `padding: 0 15px;` setting is needed not to reduce column setting buttons.
- Using padding is required to overwrite user agent stylesheet.

Reply:
Sorry, lynlynlynx is right. Reducing padding means reducing hit area, which shouldn't happen.

Reply:
I just put `padding: 0 15px`